heat/core/random.py

```diff
@@ -237,6 +239,83 @@ def __kundu_transform(values):
     return (torch.log(-torch.log(1 - values ** 0.0775)) - 1.0821) * __KUNDU_INVERSE
 
 
+def permutation(x):
+    """
+    Randomly permute a sequence, or return a permuted range.
+
+    If x is a multi-dimensional array, it is only shuffled along its first index.
+
+    Parameters
+    -----------
+    x : int or DNDarray
+        If x is an integer, call :function:`~heat.core.random.randperm`. If x is an array, make a copy and shuffle the elements randomly.
+
+    Returns
+    -----------
+    DNDarray
+
+    See Also
+    -----------
+    :function:`~heat.core.random.randperm` Random permuted range
+
+    Examples
+    ----------
+    >>> ht.random.permutation(10)
+    DNDarray([9, 1, 5, 4, 8, 2, 7, 6, 3, 0], dtype=ht.int64, device=cpu:0, split=None)
+
+    >>> ht.random.permutation(ht.array([1, 4, 9, 12, 15]))
+    DNDarray([ 9,  1, 12,  4, 15], dtype=ht.int64, device=cpu:0, split=None)
+
+    >>> arr = ht.arange(9).reshape((3, 3))
+    >>> ht.random.permutation(arr)
+    DNDarray([[3, 4, 5],
+              [6, 7, 8],
+              [0, 1, 2]], dtype=ht.int32, device=cpu:0, split=None)
+    """
+    if isinstance(x, int):
+        return randperm(x)
+
+    if not isinstance(x, dndarray.DNDarray):
+        raise TypeError("x must be integer or DNDarray")
```

Comment:
"x must be integer or DNDarray" -> "x must be an integer or DNDarray" or "x must be int or DNDarray"

Reply:
fixed